webpack/webpack.config.dev-server.js

```diff
@@ -65,6 +65,7 @@ module.exports = {
         new webpack.DefinePlugin({
           __DEVCLIENT__: false,
           __DEVSERVER__: true
-        })
+        }),
+      new webpack.IgnorePlugin(/vertx/)
```

Comment:
Exactly. If you run `npm run build:prod` you will also receive the `vertx` error